packages/component-submission/client/components/SubmissionSave.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { omit, isEqual, cloneDeep } from 'lodash'
+import { FORM_FIELDS_TO_OMIT } from '../utils/constants'
+
+class SubmissionSave extends React.Component {
+  constructor(props) {
+    super(props)
+    this.previousValues = cloneDeep(props.values)
+    this.state = {
+      isSaving: false,
+    }
+  }
+
+  componentDidMount() {
+    this.timer = setInterval(this.save, 5000)
```

Comment:
magic number ... let's use a const for now - but this would be great to be a part of the config